packages/strapi-parts/src/RawTable/RawTable.stories.mdx

```diff
@@ -14,7 +15,21 @@ import { Box } from '../Box';
 
 This is the doc of the `RawTable` component
 
-## RawTable
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/RawTable)
+
+## Imports
+
+```js
+import { RawTable } from '@strapi/parts/RawTable';
```

Comment:
`import { RawTable, RawTh, RawTd, RawTr, RawThead, RawTbody } from '@strapi/parts/RawTable';`

You can remove the imports below as they are not exported.